src/piggui.rs

```diff
@@ -110,9 +117,11 @@ fn main() -> Result<(), iced::Error> {
     }
 
     let window = window::Settings {
-        resizable: false,
-        position: window::Position::Centered,
-        size: iced::Size::new(1000.0, 900.0),
+        resizable: true,
```

Comment:
Same here: let window size be calculated (as a const) from the size of the views that it contains (their constants)